tests/PropertyTest.py

```diff
@@ -6,15 +6,46 @@
 class PropertyTest(_DataTest):
     def test_one(self):
         """ `one` should return None if there isn't a value or just the value if there is one """
-        class T(Property):
-            def __init__(self):
-                Property.__init__(self)
-                self.b = False
-
-            def get(self):
-                if self.b:
-                    yield "12"
         t = T()
         self.assertIsNone(t.one())
-        t.b=True
-        self.assertEqual('12', t.one())
+        t.b = 42
+        self.assertEqual(42, t.one())
+
+    def test_context_getter(self):
+        p = Property()
+        self.assertIsNone(p.context)
+
+    def test_context_setter(self):
```

Comment:
I think we can leave out this test... it is not a typical use case to assign the context in this way. It should be set by calling `contextualize`.

Reply:
I think I should I remove this test in ContextTest too, isn't it?